tasks/us/bls.py

```diff
@@ -1,17 +1,21 @@
 from tasks.base_tasks import (ColumnsTask, TempTableTask, TableTask, DownloadUnzipTask, TagsTask,
-                              CSV2TempTableTask, MetaWrapper)
+                              CSV2TempTableTask)
 from tasks.util import underscore_slugify, shell
 from tasks.meta import current_session, DENOMINATOR, UNIVERSE
 from tasks.us.naics import (NAICS_CODES, is_supersector, is_sector, is_public_administration,
                             get_parent_code)
 from tasks.meta import OBSColumn, OBSTag
 from tasks.tags import SectionTags, SubsectionTags, UnitTags, LicenseTags
-from tasks.us.census.tiger import GeoidColumns, SumLevel
+from tasks.us.census.tiger import GeoidColumns
 
 from collections import OrderedDict
 from luigi import IntParameter, Parameter, WrapperTask
 
 import os
+import glob
+
+
+MAX_TIMESPAN = '2017Q1'  # Example '2017Q1'
```

Comment:
Pass this as an argument to the method using it, so it's configurable from the classes using this one

Reply:
Totally agree